app/models/configuration_script_payload.rb

```diff
@@ -0,0 +1,7 @@
+class ConfigurationScriptPayload < ConfigurationScriptBase
+  # A playbook is being referenced by many job templates
```

Comment:
Yes, meant to remove. Now just for clarify. Plan to do some cleanup and rebase/squash when this discussion is more or less settled.